src/arm/96boards.c

```diff
@@ -85,6 +95,9 @@ mraa_96boards_pininfo(mraa_board_t* board, int index, int sysfs_pin, char* fmt,
     pininfo->gpio.mux_total = 0;
 }
 
+// TODO: DB820c
```

Comment:
Please remove this line. No need to mention this here

Reply:
Done